src/main/java/uk/gov/mca/beacons/service/model/AccountHolder.java

```diff
@@ -29,12 +30,16 @@
 
   private String alternativeTelephoneNumber;
 
+  @Column(name = "address_line_1")
   private String addressLine1;
 
+  @Column(name = "address_line_2")
   private String addressLine2;
 
+  @Column(name = "address_line_3")
   private String addressLine3;
 
+  @Column(name = "address_line_4")
   private String addressLine4;
```

Comment:
Override column name to match db (Hibernate was turning `addressLine1` -> `address_line1`, not `address_line_1`).  

Considered changing the migration and db column names to make Hibernate happy, but decided to match the same pattern as `BeaconPersons` table and controller etc.